docs/what-is-rocm.rst

```diff
@@ -26,57 +29,105 @@ ROCm project list
 ROCm consists of the following projects. For information on the license associated with each project,
 see :doc:`ROCm licensing <./about/license>`.
 
+Compilers
+-----------------------------------------------
+
+.. csv-table::
+  :header: "Project", "Description"
+
+  "`AOMP <https://github.com/ROCm/aomp/>`_", "Scripted build of `LLVM <https://github.com/ROCm/llvm-project>`_ and supporting software"
+  "`FLANG <https://github.com/ROCm/flang/>`_", "An out-of-tree Fortran compiler targeting LLVM"
+  "`hipCC <https://github.com/ROCm/HIPCC>`_ ", "Compiler driver utility that calls Clang or NVCC and passes the appropriate include and library options for the target compiler and HIP infrastructure"
+  ":doc:`HIPIFY <hipify:index>`", "Translates CUDA source code into portable HIP C++"
+  "`LLVM (amdclang) <https://github.com/ROCm/llvm-project>`_ ", "Toolkit for the construction of highly optimized compilers, optimizers, and runtime environments"
+
+Libraries
+-----------------------------------------------
+
+AI/ML
```

Comment:
Can we match the graphics now?  Machine Learning & Computer Vision

Reply:
Fixed